packages/xo-web/src/common/utils.js

```diff
@@ -601,3 +601,18 @@ export const getIscsiPaths = pbd => {
   const pathsInfo = pbd.otherConfig[`mpath-${pbd.device_config.SCSIid}`]
   return pathsInfo !== undefined ? JSON.parse(pathsInfo) : []
 }
+
+// ===================================================================
+
+export const downloadLog = ({ log, date, type }) => {
+  const file = new window.Blob([log], {
+    type: 'text/plain',
+  })
+  const anchor = document.createElement('a')
+  anchor.href = window.URL.createObjectURL(file)
+  anchor.download = `${new Date(date).toISOString()} - ${type}.log`
```

Comment:
This filename contains colons (`:`) which are not valid on Windows.

Reply:
Hum, um, yes, but I don't know why, on linux i have underscore `2019-02-15T17_15_27.395Z - XO.log` for example.

Reply:
It's very probable the browser translate that for you.

Reply:
Yes, you're right.